client/src/styles/main.css

```diff
@@ -1,3 +1,109 @@
 body {
-  background: red;
+  --main-bg-color: #fff;
```

Comment:
var declarations in css are not fully supported in all browsers especially not in ie. I would most likely change this later on when we come to the part where we have to check for browser comparability. maybe sass would be better?